ui/src/containers/VisualiseForm/StatementsForm/AxesEditor/CourseEditor.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component, PropTypes } from 'react';
+import { Map } from 'immutable';
+import FullActivitiesList from 'ui/containers/FullActivitiesList';
+import {
+  getTypeOpts,
+  UNIQUENESS_OPS,
+  VALUE_OPTS,
+  UNIQUENESS_VALUE_OPTS,
+} from 'ui/utils/visualisations/localOptions';
+
+export default class CourseEditor extends Component {
+  static propTypes = {
+    //type: PropTypes.string.isRequired,
+    operator: PropTypes.string,
+    value: PropTypes.instanceOf(Map),
+    model: PropTypes.instanceOf(Map),
+    updateModel: PropTypes.func.isRequired,
+    //changeOperator: PropTypes.func.isRequired,
+    //changeValue: PropTypes.func.isRequired,
+  }
+
+  static defaultProps = {
+    operator: 'uniqueCount',
+    value: UNIQUENESS_VALUE_OPTS.get('statements'),
+  }
+
+  state = {
+    operatorInput: '',
+  }
+
+  componentWillMount = () => {
```

Comment:
We should be able to remove this method since the two lines within are commented out.